tarodb/sqlc/models.go

```diff
@@ -122,6 +122,13 @@ type AssetWitness struct {
 	SplitCommitmentProof []byte
 }
 
+type AssetsMetum struct {
```

Comment:
Typo?

Reply:
Nah it does this name mangling thing to try to pluralize the table name: person goes to persons, etc. In this case, idk what it did lol (this is generated code).